### PR TITLE
[Crystal] Disable logging

### PR DESCRIPTION
**Pull request by waghanza**
Hi,

This `PR` disable logging on all `crystal` frameworks.

Logging is disabled for :

+ [x] amber
+ [x] athena
+ [x] grip
+ [x] kemal
+ [x] lucky
+ [x] orion
+ [x] router.cr
+ [x] runcobo
+ [x] shivneri
+ [x] spider-gazelle
+ [x] toro

Regards,

### CODE REVIEW

File: crystal/lucky/config/log.cr [hunk 1]
```diff
@@ -1,3 +1,3 @@
 log = Log::IOBackend.new
 log.formatter = Dexter::JSONLogFormatter.proc
-Log.dexter.configure(:info, log)
+Log.dexter.configure(:none, log)
```
Comment (jwoertink) [inline crystal/lucky/config/log.cr:3]: I think this whole file can just be

```
Log.dexter.configure(:none)
Avram::QueryLog.dexter.configure(:none)
```

Comment (waghanza) [reply crystal/lucky/config/log.cr:3]: sure, its clarify :ok_hand: 

However,  `avram` is not used

Comment (jwoertink) [reply crystal/lucky/config/log.cr]: oh, right. Good point. 👍

**Comment by waghanza** [comment]
@ujjwalguptaofficial @stakach @shootingfly @tbrand @drujensen @eliasjpr Could you give me a hand disabling logs ?

**Comment by drujensen** [comment]
They should be disabled unless you setup a config/logger.cr and configured them.  Is this not the case?

**Comment by ujjwalguptaofficial** [comment]
sure, i will send a PR @waghanza

**Comment by waghanza** [comment]
> They should be disabled unless you setup a config/logger.cr and configured them. Is this not the case?

Do you suggest to remove `config/logger.cr` to disable logging @drujensen  ? This file exists

**Comment by shootingfly** [comment]
Now runcobo v1.0.0 supports skipping request log by set ENV["SKIP_LOG"] to "true". Because of the Great Firewall of China, it is too slow to clone this project. Can you help me to commit it?

**Comment by shootingfly** [comment]
```crystal

require "runcobo"
ENV["SKIP_LOG"] = "true"

class Index < BaseAction
  get "/"

  call do
    render_plain ""
  end
end

class CreateUser < BaseAction
  post "/user"

  call do
    render_plain ""
  end
end

class ShowUser < BaseAction
  get "/user/:id"
  url NamedTuple(id: Int32)

  call do
    render_plain params[:id].to_s
  end
end

System.cpu_count.times do |_|
  Process.fork do
    Runcobo.start(reuse_port: true)
  end
end

sleep
```

upgrade to 1.0.0

**Review by stakach — APPROVED** [review]
LGTM

**Pull request merged**